kolibri/plugins/coach/assets/src/app.js

```diff
@@ -53,9 +54,23 @@ class CoachToolsModule extends KolibriModule {
         },
         {
           name: PageNames.EXAMS,
-          path: '/:class_id/exams',
+          path: '/:classId/exams',
```

Comment:
I'm personally not a fan of `underscore_case`, but @indirectlylit maintained that we should use it for now because it's closely associated with the API

https://github.com/learningequality/kolibri/pull/1122#discussion_r107735204